src/components/info/ToS.js

```diff
@@ -0,0 +1,67 @@
+
+
+export const ToSmodal = React.createClass({
+
+  getInitialState(){
+    return { showModal: true };
+  },
+
+  close(){
+    this.setState({ showModal: false });
+  },
+
+  eject(){
+    this.setState({ showModal: true });
+  },
+
+  render() {
+    return (
+      <div>
+        <Button onClick={this.open}>
+          Launch modal
+        </Button>
+
+        <Modal show={this.state.showModal} onHide={this.close}>
+          <Modal.Header closeButton>
+            <Modal.Title><img className="col-md-6" src={process.env.PUBLIC_URL + "/img/logo.png"} alt="TokenMint"  /></Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+          <div>
+            <h4>Welcome to TokenMint!</h4>
+            <br />
+            <p>This is simple to use token and ICO generator created for use with the Ethereum blockchains ETC & ETH. The purpose of this generator is to make it simple for users to create their own token and crowdsale using smart contracts. Tokens are effectively a virtual currency created and secured on top of the Ethereum blockchain. An ICO is an Intial Coin Offering. At a designated price, users can offer their tokens at a set price in either ETC or ETH. This can be useful for raising funds in a crowdsale. Tokens can be used in a variety of ways, user rewards, rebates, virtual currencies, or as a voting mechanism with a Decentralized Autonomous Organization (DAO).</p>
+            <hr />
+            <h2> Disclaimer </h2>
+            <h4>NEITHER THE SOFTWARE NOR ITS CREATORS PROVIDE LEGAL ADVICE AND THIS CODE WAS NOT CREATED TO PROVIDE LEGAL ADVICE OR AS A SUBSTITUTE FOR LEGAL ADVICE. BY USING THIS CODE YOU ALSO AGREE:</h4>
+            <ol>
+            <li> The creators of the Software and its contributors are not your lawyers.</li>
+
+            <li> The Software is not a lawyer.</li>
+
+            <li> Your use of the Software does not, in and of itself, create a legally binding contract in any jurisdiction and does not establish a lawyer-client relationship. Your communication with a non-lawyer will not be subject to the attorney-client privilege and (depending on your jurisdiction) may not be entitled to protection as confidential communication.</li>
+
+            <li> The dissemination, distribution, or usage of this software shall not constitute the provision of legal advice within your jurisdiction. Unless you are legally authorized and licensed to do so, you will not use the Software to provide or assist in the provision of legal advice.</li>
+
+            <li> You acknowledge and understand that each jurisdiction has its own particular rules regarding the practice of law. IF YOU USE THIS SOFTWARE TO PROVIDE LEGAL ADVICE YOU MAY BE SUBJECT TO CIVIL AND CRIMINAL LIABILITY. PRACTICING LAW WITHOUT A LICENSE IS A VIOLATION OF CRIMINAL LAW IN SOME JURISDICTIONS. CONSULT A LAWYER LICENSED IN YOUR JURISDICTION IF YOU HAVE ANY QUESTIONS ABOUT WHAT DOES OR DOES NOT CONSTITUTE THE PRACTICE OF LAW.</li>
```

Comment:
ha ha no as always it is borrowed from elsewhere. It is from the original dao github https://github.com/slockit/DAO